updateFromCurator.sh

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
+set -xeou pipefail
+
+# updateFromCurator.sh allows automated hash updating.
+# It is intended to be used in the curatorbin root dir.
+# VERSION_UPDATE will increment the major/minor/patch versions as specified
+## usage: ./updateFromCuratorCurator.sh ${VERSION_UPDATE}
+## usage: ./updateFromCuratorCurator.sh MAJOR
+
+NEW_HASH=$(git ls-remote git://github.com/mongodb/curator.git | head -n 1 | awk '{print $1}')
```

Comment:
nit: I would use lowercase since all caps is conventionally used for environment variables.